cdci_data_analysis/pytest_fixtures.py

```diff
@@ -402,8 +402,7 @@ def dispatcher_test_conf_with_renku_options_fn(dispatcher_test_conf_fn):
 
         f.write('\n    renku_options:'
                 '\n        renku_gitlab_repository_url: "git@renkulab.io:gabriele.barni/test-dispatcher-endpoint.git"'
-                '\n        renku_gitlab_user_name: "gabriele.barni"'
-                '\n        renku_project_url: "http://renkulab.io/projects"'
+                '\n        renku_base_project_url: "http://renkulab.io/projects"'
                f'\n        ssh_key_file: "{os.getenv("SSH_KEY_FILE", "ssh_key_file")}"')
```

Comment:
oh, could we rename this to "ssh_key_path". Else file might be read as file content.